bench-tps/src/bench.rs

```diff
@@ -314,21 +317,29 @@ fn generate_txs(
     let signing_start = Instant::now();
 
     let transactions = if let Some((
-        libra_genesis_keypair,
-        libra_pay_program_id,
+        _libra_genesis_keypair,
+        _libra_pay_program_id,
         _libra_mint_program_id,
-        libra_keys,
+        _libra_keys,
```

Comment:
Might be cleaner to `#[cfg_attr(not(feature = "move"), allow("unused_variables"))]`